azure-pipelines.yml

```diff
@@ -0,0 +1,21 @@
+# Node.js
+# Build a general Node.js project with npm.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+trigger:
+- master
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: '10.x'
+  displayName: 'Install Node.js'
+
+- script: |
```

Comment:
are we running lint and test here?

Reply:
Is this just running install for the parent? how does the mono repo stuff play here?

Reply:
We would want to add a post install hook to the root package to run `lerna bootstrap` to have all the subpackages be installed correctly.

Reply:
As far as I know, this one is also unintentional PR opened when @pkanher617 is trying some Azure pipelines integration in our code. FYI.